examples/angular2/tsconfig.json

```diff
@@ -1,15 +1,13 @@
 {
   "compilerOptions": {
-    "target": "ES5",
+    "target": "es5",
     "module": "commonjs",
     "moduleResolution": "node",
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": true
-  },
-  "files": [
-    "app/bootstrap.ts"
-  ]
+    "noImplicitAny": true,
+    "suppressImplicitAnyIndexErrors": true
+  }
```

Comment:
For simplicity there's no need for do this, the TS files for the developer and the JS files for the browser. Do any developer will care the output JS files?